src/main/resources/openapi/EducationAndWorkPlanAPI.yml

```diff
@@ -0,0 +1,278 @@
+openapi: 3.0.0
+info:
+  title: Education and Work Plan API
+  version: '1.0.0'
+  description: Education and Work Plan API
+  contact:
+    name: Matt Wills
+    email: matt.wills@digital.justice.gov.uk
+servers:
+  - url: 'http://localhost:3000'
+#
+# --------------------------------------------------------------------------------
+#
+paths:
+  #
+  # --------------------------------------------------------------------------------
+  # RESTful style endpoints start here
+  # --------------------------------------------------------------------------------
+  #
+
+  #
+  # Goals
+  # --------------------------------------------------------------------------------
+  '/prisoners/{prisonNumber}/action-plan':
+    parameters:
+      - name: prisonNumber
+        description: The ID of the Prisoner (confusingly not the prison) who may have created (or is in the process of creating) an "Action Plan", consisting of one or more Goals.
+        schema:
+          type: string
+        example: 'A1234BC'
+        in: path
+        required: true
+    get:
+      summary: Return a Prisoner's Action Plan
+      description: Return a Prisoner's Action Plan
+      tags:
+        - Action Plan
+      responses:
+        '200':
+          $ref: '#/components/responses/ActionPlan'
+      operationId: get-action-plan-by-prison-number
+
+  '/prisoners/{prisonNumber}/action-plan/goals':
+    parameters:
+      - name: prisonNumber
+        description: The ID of the Prisoner (confusingly not the prison) who may have created (or is in the process of creating) an "Action Plan", consisting of one or more Goals.
+        schema:
+          type: string
+        example: 'A1234BC'
+        in: path
+        required: true
+    post:
+      summary: Creates a Goal.
+      description: Creates a Goal and saves it to the Prisoner's Action Plan (which will be created if it doesn't already exist).
```

Comment:
What if it does already exist? How do we tell if it exists or not (equality)? Do we care? or do we just create it?

Reply:
I assume we just create it. We could have dozens of goals that are effectively duplicates, but I don't think it would ever be used that way.

Reply:
Agreed. Which was why I was wondering why we even mention it - ie. whats the relevance/significance of "which will be created if it doesn't already exist" ?
We are going to simply create it?  no?

Reply:
I was referring to the ActionPlan. I'll try to make it clearer.